Editor/AGS.Editor/Components/AudioComponent.cs

```diff
@@ -497,6 +503,50 @@ private void SetActualAudioClipProperties(AudioClip clip, int inheritedVolume, b
             }
         }
 
+        private void CommandForceReimportOfAudioClip(AudioClip clip)
+        {
+            if(!ForceReimportOfAudioClip(clip))
+            {
+                _guiController.ShowMessage("Failed to reimport Audio Clip "  + clip.ScriptName + ".", MessageBoxIconType.Warning);
+            }
+        }
+
+        private bool ForceReimportOfAudioClip(AudioClip clip)
+        {
+            string sourceFileName = clip.SourceFileName;
+            string cacheFileName = clip.CacheFileName;
+            bool reimportOk = false;
+            if(File.Exists(sourceFileName) && !string.IsNullOrEmpty(cacheFileName))

```

Comment:
This whole code from here and until the end of the function looks like something that would be better as a generic method "try to replace a file and fallback to backup if failed". There's [Utilities class](https://github.com/adventuregamestudio/ags/blob/master/Editor/AGS.Editor/Utils/Utilities.cs) that might be a good place to put such method (alternatively there could be FileUtilities class, but that's a separate refactor task).